debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -1220,6 +1220,11 @@ STRING_USER_NAME:                    (
                                      (
                                        SQUOTA_STRING | DQUOTA_STRING 
                                        | BQUOTA_STRING | ID_LITERAL
+                                       | IP_ADDRESS
```

Comment:
IP_ADDRESS | IP6_ADDRESS ?

Reply:
Does MySQL support that?

Reply:
yep. and #1977 has already lexer code for that. Maybe we close this one in favor of #1977?

Reply:
In fact I was hoping we could go for this one, as it's much less complex.  Keep in mind we don't really have any interest in processing these statements, essentially it's good enough if the parser doesn't choke on them.